### DISCUSSION TITLE
Google Pixel 8a connectivity issue with XDrip

### DISCUSSION THREAD

**Discussion by IrineLTU** [NightscoutFoundation/xDrip#3639] [Bug reports]
Hi, i started mu Insulin Closed Loop with my Dana Pump, Dexcom One, Google Pixel 8a and AAPS / XDrip apps. All went well for first 12 hours with date reception and pump control. But suddenly XDrip lost connection to Dexcom via Bluetooth. Although Pump was still connected. I tried do restart sessions by changing the sensor name (making it incorrect), restarting phone, correcting the sensor’s name and restating session. It received date once from Dexcom and later connection was lost again. I tried to change BlueTooth AVRCO developer settings to 1,3 - but it did not help. 
I thought maybe it is a sensor issue.
But also was reading that Pixel 8a has some bluetooth issues as well. Now I have a new Dexcom One, but not sure is it a phone or a sensor, or an app issue. Please advise. 

https://github.com/user-attachments/assets/d7a256f5-2601-42ce-b81d-0338108b67f9

![IMG_3692](https://github.com/user-attachments/assets/a29171da-8099-4a3c-b051-f3331f7222a8)
![IMG_3691](https://github.com/user-attachments/assets/4527d128-e7ac-4d22-9a1e-acb1a9ef7514)

**Comment by Navid200** [comment]
Have you disabled xDrip battery optimization?
Have you disabled Bluetooth battery optimization?

**Reply by IrineLTU** [reply]
Yes, I did. Did not helped.

**Reply by Navid200** [reply]
It doesn't look like you have followed the directions here: https://navid200.github.io/xDrip/docs/Dexcom/G7.html

**Reply by IrineLTU** [reply]
Well. I recheck all steps and to me it looks what I did all steps as You mentioned. Due to stressful situations I manualy (without stoping on X Drip) removed old sensor and waited for several days fif new DecCom to arrive (before Used MedTronic pump and CHM). Now I feel confused about starting a new one - should I pair a new one as a replacement of old, should I pair it totally as a new one, or reinstall XDrip at all (no data collected yet). If to use last option - should I reinstall AAPC. Many questions- no answer:( Last several nights without any CGM were as a nightmare. Thanks a lot.

**Comment by jamorham** [comment]
Can you just clarify - Which type of sensor was this with? Is it working now or?

**Reply by IrineLTU** [reply]
Dexcom One. I removed the sensor phisicaly (without stoping by XDrip) as it was not funcioning. Now I am in doubt that was the reason for malfunction - sensor or the phone. And the answer for this question dictates me next steps - to change the phone or restart new sensor, which comes with challenge mentioned above.

**Comment by Navid200** [comment]
The video clip you have posted shows no last connected.  That is a transmitter connectivity issue.
The screenshot you have posted shows deep sleeping error.  That is a transmitter connectivity issue as well.
Neither of those can be caused by how you stopped a previous sensor.  

But, I am afraid you are mixing up sensor and transmitter in your text.  And that's why we have a hard time proving advice.
  
When you say you now have a new sensor, do you mean a new transmitter or a new sensor?
You should be able to connect to a used transmitter without a sensor.  In fact, you need to do that before using a new sensor to avoid possibly wasting a sensor.  

On the other hand, if you are talking about a new transmitter, you said that it  was working with your pump.  Then, it is likely there was nothing wrong with the transmitter and you may needed to try a different phone or some settings were wrong.

Can you clarify if you have a new transmitter or a new sensor?

**Reply by Navid200** [reply]
Aren't you supposed to use a Dana pump with CamAPS?

**Reply by Navid200** [reply]
If you are using CamAPS, you will have to use xDrip in Companion App mode unfortunately.

**Reply by Navid200** [reply]
If you are using CamAPS, it uses the mobile app Bluetooth slot of the transmitter.  So, you will not be able to use Dexcom app or xDrip or any other mobile app.

xDrip has a feature that gets readings from a companion app in this case, CamAPS.
In contrast, t:slim is a pump that follows protocol and uses the medical slot of the transmitter.  Therefore, if you were using that pump, you would be  able to use either Dexcom app or xDrip.  

If you are stuck with CamAPS,  which is an app that does not follow protocol, you will be limited to the companion app mode: 
https://navid200.github.io/xDrip/docs/Follow/CompanionApp.html

**Reply by IrineLTU** [reply]
First of all, apologies for late reply. I started a freshly new DexCom One sensor with transmiter and disabled Google Play Updates. It all went well - so my conclusion - Google Pixel 8a works well. As always - main challenge are with phone settings. As the phone is new, so it has more restrictions and safety features, which blocks signal to XDrip. Thanks again for You support.

**Reply by Navid200** [reply]
Do you mean Google Play Protect?

If yes, that's interesting.  The only issue we have seen with Google Play Protect is with some phones disabling connectivity once a day exactly at the same time.
I have never experienced it with my Pixel 6a though.

Thanks for reporting this.  It will be great if you can confirm that you meant Google Play Protect or something else.
